static/skin/index.js

```diff
@@ -453,6 +454,24 @@
         }
     }
 
+    function updateNavVisibilityState() {
+        const st = window.scrollY;
+        const scrollPossible = document.querySelector('.kiwixHomeBody').offsetHeight > window.screen.height;
```

Comment:
Isn't it better to use `window.innerHeight` instead of `window.screen.height`? After all what matters is the size of the window rather than the size of the screen. Then this functionality will work correctly also in small non-maximized windows on large screens (the way I tested this PR on my computer).

Reply:
On the other hand why we need this `scrollPossible` variable at all?

Reply:
There was a blink effect on trying to scroll when only 1 book was shown (it kept removing and adding kiwixNav). No idea how I can't reproduce it anymore. Removed this variable